ssl/statem/extensions_clnt.c

```diff
@@ -108,8 +108,8 @@ static int use_ecc(SSL *s)
     return i < end;
 }
 
-int tls_construct_ctos_ec_pt_formats(SSL *s, WPACKET *pkt, X509 *x,
-                                     size_t chainidx, int *al)
+int tls_construct_ctos_ec_pt_formats(SSL *s, WPACKET *pkt, unsigned int context,
```

Comment:
Six lines of assignment in the common calling function vs pushing six values onto the stack every single time.  Not that this matters, it's such a micro-efficiency issue.

This is about code readability and maintainability.  Most functions don't need all parameters, and the #2270 (unused-arg) branch has many many ```(void)``` casts that wouldn't be needed.  As for maintainability, you've already added three parameters, and it's not clear you're done yet :)  And also for follow-on work, it's much less invasive to change one struct then to add a param everywhere.